packages/core/test/insured-bridge/e2e/InsuredBridge.e2e.js

```diff
@@ -47,67 +60,132 @@ describe("L1 <> L2 Deposit and Withdrawal", () => {
     });
 
     // Deploy an ERC20 token on L1. Add minting to the l1Wallet and mint some tokens.
-    L1_ERC20 = await factory__L1_ERC20.connect(l1Wallet).deploy("L1 ERC20 Token", "L1Tkn", 18);
-    await L1_ERC20.deployTransaction.wait();
+    l1Erc20 = await factory__L1_ERC20.connect(l1Wallet).deploy("L1 ERC20 Token", "L1Tkn", 18);
+
+    await l1Erc20.deployTransaction.wait();
 
-    await L1_ERC20.addMember(1, l1Wallet.address);
-    await L1_ERC20.mint(l1Wallet.address, 1234);
+    await l1Erc20.addMember(1, l1Wallet.address);
+    await l1Erc20.mint(l1Wallet.address, 1234);
 
     // Deploy the paired ERC20 token to L2. This takes in the address of the L2 bridge and the associated L1 token.
-    L2_ERC20 = await factory__L2_ERC20
```

Comment:
updated some of these variables to be consistent with the rest of UMA unit test naming convention.